nicolegooden/potter
# Homepage

#### What's this PR do?  

+ Adds styling to `HomePage` component
+ Organizes files for `HomePage` and `App` components via specific directories
+ Allows user to experience being sorted randomly into one of the four houses
+ Allows user to see which house was chosen for them
+ Allows user to see two new buttons to learn more about their house or to select a character (currently not functional, just for display)
+ Adds prop types for `Homepage`

#### How should this be manually tested?  

Open the app and click `find my house` - should see message and button text change to indicate a house has been selected, as well as a button to choose a character

#### Any background context you want to provide?  

N/A

#### What are the relevant tickets?  

Closes #1 
Closes #2 

#### Screenshots (if appropriate) 

<img width="1234" alt="Screen Shot 2020-11-06 at 6 16 23 PM" src="https://user-images.githubusercontent.com/62262404/98427882-4a444400-205c-11eb-8570-6e33855e0c97.png">

<img width="1195" alt="Screen Shot 2020-11-06 at 6 16 36 PM" src="https://user-images.githubusercontent.com/62262404/98427887-4d3f3480-205c-11eb-8117-97030da14764.png">
